components/hero-unit.jsx

```diff
@@ -34,6 +34,12 @@ var HeroUnit = React.createClass({
         <div className="col-md-12 hero-unit" style={{
           backgroundImage: 'url(' + this.state.image + ')'
         }}>
+          <marquee>
+            <span style={{color: 'transparent'}}>
+              MOZILLAAAAAAAAA!!
+            </span>
```

Comment:
For some reason this `<span>` is needed in Firefox for the `<img>` below it to show. Regardless, it provides useful semantic content for screen readers and search engines.